src/devices/Sht3x/samples/Program.cs

```diff
@@ -14,10 +14,10 @@
 while (true)
 {
     Temperature tempValue = sensor.Temperature;
-    Ratio humValue = sensor.Humidity;
+    RelativeHumidity humValue = sensor.Humidity;
 
     Console.WriteLine($"Temperature: {tempValue.DegreesCelsius:0.#}\u00B0C");
-    Console.WriteLine($"Relative humidity: {humValue:0.#}%");
+    Console.WriteLine($"Relative humidity: {humValue:0.#}");
```

Comment:
Maybe humValue.Percent?

Reply:
```suggestion
    Console.WriteLine($"Relative humidity: {humValue.Percent:0.#}%RH");
```

Reply:
humValue will also format correctly likely but not sure how many digits after comma (it will print `<value>%RH`)